.eslintignore

```diff
@@ -0,0 +1 @@
+addon/ng2/blueprints/ng2/files/src/config/system.config.js
```

Comment:
Because of the double quotes which are mandatory for `JSON.parse` in `systemjs-helper.ts`.